clang/test/Sema/attr-assume.c

```diff
@@ -1,14 +0,0 @@
-// RUN: %clang_cc1 -triple i386-apple-darwin9 -fsyntax-only -verify %s
```

Comment:
Same here, can we just rename this to attr-omp-assume.c and change the name in the test?